v1/pipeline/steps/ci/resource.yaml

```diff
@@ -0,0 +1,13 @@
+title: Resource
+type: object
```

Comment:
we already have resource.yaml and resources.yaml in v1/pipeline folder, can we use them?